spidermon/contrib/actions/sentry/__init__.py

```diff
@@ -100,15 +102,47 @@ def get_message(self):
 
         return message
 
+    @staticmethod
+    def get_tags(message):
+        tags = {}
+        failed_monitors = message.get("failed_monitors", [])
+        failed_monitors = [y.split("/") for y in failed_monitors]
+        for key, group in groupby(failed_monitors, key=lambda x: x[0]):
+            for mon in group:
+                try:
+                    k = mon[1].lower().replace(" ", "_")
+                    # 32 chars undocumented tag key length
```

Comment:
**question**: Is this a limit from Sentry (not allowing tags with more than 32 chars)? If so, maybe it is worthy to include that information in the documentation of the action.

Reply:
Actually, it's now [documented](https://develop.sentry.dev/sdk/data-handling/):

```
Tag keys are limited to 32 characters.
Tag values are limited to 200 characters.
```